app/src/main/java/de/danoeh/antennapod/config/StorageCallbacksImpl.java

```diff
@@ -125,6 +125,7 @@ public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
                     PodDBAdapter.KEY_CHAPTER_TYPE));
         }
         if(oldVersion <= 14) {
```

Comment:
Concerning *mark all as read*: There is an option in the feed overview when at least one of the episodes is unplayed.